package.json

```diff
@@ -7,8 +7,8 @@
 		"tests/e2e"
 	],
   "engines": {
-    "node": "22.x.x",
-    "npm": "10.x.x"
+    "node": "24.x.x",
+    "npm": "11.x.x"
```

Comment:
You can't upgrade node like this. It requires other changes (CI, Docker image, etc..)

Reply:
Reverted for now: https://github.com/writer/writer-framework/pull/1158/commits/0564ef33b9e11f36fecafb9a750d0b09d00aefb3